support/make/install.mk

```diff
@@ -1,5 +1,6 @@
 install: ## Install gem dependencies
 	@echo "Installing gem dependencies"
+	@gem install bundler -v '~> 1'
```

Comment:
@pawelpacana This may be controversial but allows to build consistently.

Our constraints are:

* A `Gemfile.lock` should exist
* Should we bundle with bundler `~> 2` we end up in a dependency conflict on the Rails 4.2 facet of your build.
* The `Gemfile.lock` only matters for RES developers, not users of releases.
* Bundler `2.x` can if `1.x` is installed fall back to 1.x.

So unless you are willing to drop the build for Rails 4.2, this seems to be the common denominator. 

If this where my project, I'd try to install the bundler at the correct version directly in the project specific docker image. But for now I'm going path of least resistance.

OT: Adding the `Gemfile.lock` makes the odd mutations I was trying to debug disappear. I've to assume at this point it was dependency indeterminism which `Gemfile.lock` are made to prevent.

Reply:
Rails 4.2 support might be quite possible to drop on `master` branch

Reply:
I'd love that. But for the moment I can reproduce an odd result without the Gemfile.lock here also.

I'm happy to extract the gemfile lock addition in a separate PR.

But my main focus is to reproduce the odd result here. I can confirm the new coverage criteria improve the situation, but I'm not 100% happy (yet).